egui-winit/src/window_settings.rs

```diff
@@ -2,22 +2,32 @@
 #[derive(Clone, Copy, Debug)]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
 pub struct WindowSettings {
-    /// Inner position of window in physical pixels
-    inner_pos: Option<egui::Pos2>,
+    /// Position of window in physical pixels. This is either
+    /// the inner or outer position depending on the platform.
+    position: Option<egui::Pos2>,
     /// Inner size of window in logical pixels
     inner_size_points: Option<egui::Vec2>,
 }
 
 impl WindowSettings {
     pub fn from_display(window: &winit::window::Window) -> Self {
         let inner_size_points = window.inner_size().to_logical::<f32>(window.scale_factor());
-
-        Self {
-            inner_pos: window
+        let position = if cfg!(macos) {
+            // Use the inner position for MacOS
```

Comment:
I think, it is better to wording like "MacOS uses inner position when positioning windows" or something, added details from your findings also would be helpful.

Right now this comment looks like Captain Obvious statement.